### ISSUE TITLE
Large files

### ISSUE THREAD

**Issue by aolofsson** [idea-fasoc/OpenFASOC#70]
Perhaps not a big issue, but the repo is >800M...seems excessive. Might be worth a large file clean (saw a large mp4 in there for example): repos/OpenFASOC/tapeouts/mpw-1/testsetup/MeasResults/SanityCheckDemo.mp4

**Comment by msaligane** [comment]
Thanks @aolofsson 
We will address that!

**Comment by luccareinehr** [comment]
@msaligane maybe create a separate repo "openfasoc-tapeouts" for the tapeouts?

This way people who want to use openfasoc get only the tool when cloning the repository (and not other test files)

**Comment by msaligane** [comment]
Good idea @luccareinehr do you want to address it?

**Comment by luccareinehr** [comment]
@msaligane Yes sure, but I can't create repos in the idea-fasoc organization. 

Could you create one named openfasoc-tapeouts? Then I can submit a pull request there

**Comment by msaligane** [comment]
@luccareinehr I created a new repo: https://github.com/idea-fasoc/openfasoc-tapeouts
Can you please move the files there.

**Comment by luccareinehr** [comment]
@msaligane I noticed most of the remaining repository size (~400 MB) is due to the [common/platforms/](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/common/platforms directory) folder. For now, I don't know what we should do about that

**Comment by msaligane** [comment]
I think that could be optimized as well. Let's discuss that with @proppy

**Comment by saicharan0112** [comment]
@msaligane are generators using/ planned to use any of the osu libraries listed here? [common/platforms/](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/common/platforms)

**Comment by msaligane** [comment]
we use them for the cryo-gen. We should find a way to have them as submodules or fetch them directly from open-pdks..

**Comment by saicharan0112** [comment]
Sounds good. We can have them in a seperate repo similar to the testcases (https://github.com/idea-fasoc/openfasoc-tapeouts). Let's work on it when we get some time.

**Comment by Myrausman** [comment]
I want to work on it , can you please create the repo for this [common/platforms/](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/common/platforms) or share the link of it, if already made

**Comment by msaligane** [comment]
@Myrausman That would be great! Here you go: https://github.com/idea-fasoc/platforms
We have a Friday weekly meeting you could join if you have cycles. Please let me know!

**Comment by Myrausman** [comment]
@msaligane I'm really sorry for delayed response, thank you for creating the repo , we can do a quick meeting if you suggest.

**Comment by msaligane** [comment]
Please email me at: mehdi@umich.edu
